packages/flutter_tools/templates/create/pubspec.yaml.tmpl

```diff
@@ -8,3 +9,37 @@ dev_dependencies:
   flutter_driver:
     sdk: flutter
 {{/withDriverTest}}
+
+# For information on the generic Dart part of this file, see the
+# following page: https://www.dartlang.org/tools/pub/pubspec
+
+# The following section is specific to Flutter.
+flutter:
+
+  # The following line ensures that the Material Icons font is
+  # included with your application, so that you can use the icons in
+  # the Icons class.
+  uses-material-design: true
+
+  # To add assets to your application, add an assets section here, in
+  # this "flutter" section, as in:
+  # assets:
+  #  - images/a_dot_burr.jpeg
+  #  - images/a_dot_ham.jpeg
```

Comment:
To whomever did this, I see you 👀 
![hamilton the musical scene](https://66.media.tumblr.com/0ebb0fd083d9fc9f986a448924e621f5/tumblr_njy8tfCm8L1qk6b52o1_250.gifv)